golem-templates/templates/ts/ts-app-component-default/components-ts/component-name/src/main.ts

```diff
@@ -1,18 +1,18 @@
-import {ComponentNameApi} from "./generated/component-name";
+import type * as World from "pack:name/component-name"
```

Comment:
do they recommend using the "World" name, or this is something we want to (given worlds do not really "exists" art this point)? I think usually in other tools we just use binding.

Reply:
Let's use binding then